app/views/vms/_vm_status.html.erb

```diff
@@ -1,17 +1 @@
-<div class="round <%= vm.powered_on? ? 'bg-success' : 'bg-warning mt-1'  %>"></div>
-<% if current_user.admin? || vm.sudo_users.include?(current_user) %>
-  <% if vm.vm_ware_tools? %>
-    <%= link_to '',
-                {controller: :vms, action: 'change_power_state', id: vm.name},
-                method: :post,
-                data: { confirm: 'Are you sure?' },
-                class: "btn-manage float-right #{ vm.powered_on? ? 'stop' : 'play' }" %>
-    <% if vm.powered_on? %>
-      <%= link_to '',
-                  {controller: :vms, action: 'reboot_guest_os', id: vm.name},
-                  method: :post,
-                  data: { confirm: 'Are you sure?' },
-                  class: "btn-manage float-right replay" %>
-    <% end %>
-  <% end %>
-<% end %>
+<div class="round <%= vm.archived? ? 'bg-secondary' : (vm.powered_on? ? 'bg-success' : 'bg-danger') %>"></div>
```

Comment:
Most likely don't need a separate partial for a single line of code

Reply:
Fixed in https://github.com/hpi-swt2/vm-portal/pull/458/commits/689a0371a659cfda7d3a22b64bf117d3ff987950